### DISCUSSION TITLE
Machine frame placement

### DISCUSSION THREAD

**Discussion by Rearth** [Rearth/Oritech#17] [Todos]
Currently, machine frames need to be placed by hand, and the inner area also needs to be cleaned manually. A machine to do that task would be great. 

The Idea is that is has a small UI where the size is selected and a preview can be turned on. There's a small inventory for machine frames.

Does not need energy to work. It just places the machine frames and clears the inner area afterwards without dropping the harvested items (they are lost)

**Comment by jshipley** [comment]
It would be nice if the machine frame machine could also dismantle machine frames. It would be really useful for quarrying. You could set it up over a chunk, mine out the entire chunk, and then make it break the machine frames and pull them back into the inventory the machine frames came from (maybe a 1-2 slot internal inventory just for machine frames).
